src/components/tabs/js/tabsController.js

```diff
@@ -684,6 +684,17 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
     for (newIndex = index + inc;
          ctrl.tabs[ newIndex ] && ctrl.tabs[ newIndex ].scope.disabled;
          newIndex += inc) {}
+
+    // Handle cycling through the tabs
+    var lastIndex = ctrl.tabs.length - 1;
+    // If the left arrow is pressed from the first tab
+    if (inc === -1 && index === 0) {
+      newIndex = lastIndex;
+    // If the right arrow is pressed from the last tab
+    } else if (inc === 1 && index === lastIndex) {
+      newIndex = 0;
+    }
+
```

Comment:
You removes the unit test checks for no-op on the ends, but didn't add new checks for the wrapping behavior.